src/coreclr/jit/morph.cpp

```diff
@@ -11581,6 +11581,14 @@ GenTree* Compiler::fgMorphSmpOp(GenTree* tree, MorphAddrContext* mac)
             break;
     }
 
+    // Perform strength reduction prior to preorder op1, op2 operand processing
```

Comment:
Nit: I am not convinced adding the comment here has a lot of value, it is fairly obvious from the code what is going on.